csrc/scheduler/vectorize_helper.cpp

```diff
@@ -477,17 +483,14 @@ ContiguousInnerDimensionsMapper::computeInfoC2P(
         clear_pos = (int)i;
       }
     }
-    // Clear everything to the left of the inner most resolved broadcast
-    // dimension, including the broadcasted domain.
-    if (clear_pos >= 0) {
-      consumer_ids_to_clear.insert(
-          from->getRootDomain().begin(),
-          from->getRootDomain().begin() + clear_pos + 1);
-    }
   }
 
   std::vector<IterDomain*> producer_rfactor_ids;
-  for (auto from_id : from_ids) {
+  for (int64_t i : c10::irange(from_ids.size())) {
```

Comment:
```suggestion
  for (int64_t i : c10::irange(clear_pos, from_ids.size())) {
```

Reply:
I guess I really needed some sleep at this point :laughing: